public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
I've never seen the meta name='viewport' before. I'll have to start using that.

Reply:
lol i think that might have been from some boilerplate html shortcut thing when i type in html5. i'll definitely have to understand what it does.